docs/multiindex_isotope_decay_data.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    atom_data_file = '/Users/anirbandutta/Software/tardis-refdata/atom_data/kurucz_cd23_chianti_H_He.h5'
</pre><p>Would be better to point to the default atom data directory or a relative path perhaps</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>